Laba1-0/src/main/kotlin/Laba6/3.kt

```diff
@@ -7,7 +7,7 @@ package Laba6
  * */
 
 fun main() {
-    var s: String? = null
-    s = s ?: "empty"
-    println(s)
+    val s: String? = null
+    val m: String? = s ?: "empty"
```

Comment:
```suggestion
    val m: String = s ?: "empty"
```
но вообще можно не указывать

Reply:
Да. Вопрос в том нужен ли нам nullable m или нет. Ок, поправила